lib/node_modules/@stdlib/blas/base/dnrm2/test/test.ndarray.native.js

```diff
@@ -62,6 +93,31 @@ tape( 'the function calculates the L2-norm of a vector', opts, function test( t
 	z = dnrm2( x.length, x, 1, 0 );
 	t.strictEqual( z, 4.0, 'returns expected value' );
 
+	x = new Float64Array( [ 1.0e150, 1.0e150, 1.0e150, 1.0e150 ] );
+
+	z = dnrm2( x.length, x, 1, 0 );
+	t.strictEqual( z, 2.0e+150, 'returns expected value' );
+
+	x = new Float64Array( [ 1.0e-155, 1.0e-155, 1.0e-155, 1.0e-155 ] );
+
+	z = dnrm2( x.length, x, 1, 0 );
+	t.strictEqual( z, 2.0e-155, 'returns expected value' );
+
+	x = new Float64Array( [ 1.0e150, 1.0e50, 1.0e150, 1.0e50 ] );
+
+	z = dnrm2( x.length, x, 1, 0 );
+	isApprox( t, z, 1.41421e150, 2.0 );
```

Comment:
@ShabiShett07 I believe you still need to adjust the precision in this file, as well, correct?